Lib/unittest/mock.py

```diff
@@ -820,7 +820,16 @@ def _get_call_signature_from_name(self, name):
                 break
             else:
                 children = child._mock_children
```

Comment:
Note to self, reviewer : Do we need to default to empty dictionary too guarding against `AttributeError` for `_mock_children`?

Reply:
What test cases could we add to verify that?